src/libcore/option.rs

```diff
@@ -208,6 +208,30 @@ impl<T> Option<T> {
         !self.is_some()
     }
 
+    /// Returns `true` if the option is a [`Some`] value containing the given value.
+    ///
+    /// # Examples
+    ///
+    /// ```
```

Comment:
```suggestion
    /// ```
    /// #![feature(option_contains)]
    ///
```